### DISCUSSION TITLE
Reorder screens

### DISCUSSION THREAD

**Discussion by hannesfant** [Budibase/budibase#3624] [Help]
Is there any way to reorder screens/routes? Currently it looks like the app defaults to the first screen/route in the screens list, which isn't ideal since my first screen happens to be a page for creating records, instead of the "main dashboard"

**Comment by joebudi** [comment]
Hey @hannesfant 

If you navigate to the main dashboard screen settings and change the root to `/` it should make your main dashboard the home screen. You will need to update any links pointing to your main dashboard due to the route change.
![CleanShot 2021-12-01 at 17 34 09](https://user-images.githubusercontent.com/49767913/144284436-dee42f22-73dc-4d1a-b088-5d8cb4fe5ac1.png)
I hope that helps.
